javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,145 @@
+const name= [];// use to save the name of  Benjamin
+
+const questions={ // start of Object of questions
+    
+    question1:"Hello my name is Benjamin",
+    question2:"I need information regarding to hackyourfuture",
+    question3:"do you address of hackyourfuture ",
+    question4:"Add fishing to my todo",
+    question5:"Remove fishing from my todo",
+    question6:"What is on my todo?",
+    question7:"What day is it today?",
+    question8:"what is 3 + 3",
+    question9:"Set a timer for 4 minutes",
+       
+}
+
+const answers={ // start of  Object of answers 
```

Comment:
Normally when you have a variable name that is plural (multiple), then you would expect an array, because that is how you store multiple values